dts/arm/st/f4/stm32f429Xi.dtsi

```diff
@@ -24,4 +25,44 @@
 			};
 		};
 	};
+
+	sdram2: sdram@d0000000 {
+		device_type = "memory";
+		status = "disabled";
+		reg = <0xd0000000 DT_SIZE_M(8)>;
+	};
+
+	fmc: memory-controller@a0000000 {
+		compatible = "st,stm32-fmc";
+		reg = <0xa0000000 0x1000>;
+		clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00000001>;
+		label = "STM32_FMC";
+		status = "disabled";
+
+		fmc_sdram: fmc_sdram {
+			compatible = "st,stm32-fmc-sdram";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			label = "STM32_FMC_SDRAM";
+			status = "disabled";
+
+			power-up-delay = <100>;
+			num-auto-refresh = <8>;
+			mode-register = <0x220>;
+			refresh-rate = <1386>;
+
+			bank@1 {
+				reg = <1>;
+
+				st,sdram-control = <
+					STM32_FMC_SDRAM_NC_8 STM32_FMC_SDRAM_NR_12
+					STM32_FMC_SDRAM_MWID_16 STM32_FMC_SDRAM_NB_4
+					STM32_FMC_SDRAM_CAS_2 STM32_FMC_SDRAM_SDCLK_PERIOD_2
+					STM32_FMC_SDRAM_RBURST_ENABLE STM32_FMC_SDRAM_RPIPE_0>;
+				st,sdram-timing = <2 7 4 7 2 2 2>;
```

Comment:
SoC should not contain such defaults, they can take any value depending on the connected memory/es.

Reply:
Ok that makes sense. Should it be in stm32f429i_disc1.dts? (moved it there in last commit)